QtTools: Skip unnecessary dependencies:

1. The LLVM installatiion.
2. Build only required tools.


--- a/qttools/configure.cmake
+++ b/qttools/configure.cmake
@@ -6,8 +6,6 @@ include(${CMAKE_CURRENT_LIST_DIR}/src/qdoc/cmake/QDocConfiguration.cmake)
 
 #### Tests
 
-qt_find_package(WrapLibClang 8 PROVIDED_TARGETS WrapLibClang::WrapLibClang)
-
 if(TARGET WrapLibClang::WrapLibClang)
     set(TEST_libclang "ON" CACHE BOOL "Required libclang version found." FORCE)
 endif()

--- a/qttools/src/linguist/CMakeLists.txt
+++ b/qttools/src/linguist/CMakeLists.txt
@@ -9,16 +9,8 @@ if(NOT QT_FEATURE_linguist)
     return()
 endif()
 add_subdirectory(lconvert)
-add_subdirectory(lprodump)
 add_subdirectory(lrelease)
-add_subdirectory(lrelease-pro)
 add_subdirectory(lupdate)
-add_subdirectory(lupdate-pro)
-if(QT_FEATURE_process AND QT_FEATURE_pushbutton AND QT_FEATURE_toolbutton
-    AND QT_FEATURE_png AND QT_FEATURE_mdiarea AND QT_FEATURE_syntaxhighlighter
-    AND TARGET Qt::Widgets AND TARGET Qt::Quick)
-    add_subdirectory(linguist)
-endif()
 
 # Create a fake module that would emulate the Qt5::LinguistTools CMake Config package
 qt_internal_add_module(Linguist
